app/services/induction/change_preferred_email.rb

```diff
@@ -3,7 +3,7 @@
 class Induction::ChangePreferredEmail < BaseService
   def call
     Induction::ChangeInductionRecord.call(induction_record:,
-                                          changes: { preferred_identity: })
+                                          changes: { preferred_identity:, end_date: nil })
```

Comment:
What if this IR had a future `end_date` e.g. the participant was transferring, this would remove their `end_date`?  Same issue with the `ChangeMentor`